shardingsphere-test/shardingsphere-rewrite-test/src/test/resources/scenario/mix/case/select_for_query_with_cipher.xml

```diff
@@ -56,14 +56,14 @@
     
     <rewrite-assertion id="select_with_sharding_qualified_shorthand_join_table">
         <input sql="SELECT b.* FROM t_account a, t_account_detail b where a.password = b.password" />
-        <output sql="SELECT b.* FROM t_account_0 a, t_account_detail_0 b where a.assisted_query_password = b.password" />
-        <output sql="SELECT b.* FROM t_account_1 a, t_account_detail_1 b where a.assisted_query_password = b.password" />
+        <output sql="SELECT b.* FROM t_account_0 a, t_account_detail_0 b where a.assisted_query_password = b.assisted_query_password" />
```

Comment:
@cheese8 I think this pr is a big change, can we add more test case for this pr?